### PR TITLE
Added --details option to totp command

### PR DESCRIPTION
**Pull request by dstromberg**

### CODE REVIEW

File: keepercommander/commands/recordv3.py [hunk 1]
```diff
@@ -1768,6 +1770,35 @@ def __init__(self, record_uid, record_title, paths):
         self.paths = paths
 
 
+def extract_otp_url(data):
```
Comment (sk-keeper) [inline keepercommander/commands/recordv3.py]: It looks like this code does not work for legacy records

Comment (sk-keeper) [reply keepercommander/commands/recordv3.py]: totp --details  
do not make any difference for me

Comment (dstromberg) [reply keepercommander/commands/recordv3.py]: > It looks like this code does not work for legacy records

True - it currently makes no attempt to deal with legacy records.  Should it?

Comment (dstromberg) [reply keepercommander/commands/recordv3.py]: > totp --details
> do not make any difference for me

You have to specify a record.

Comment (dstromberg) [reply keepercommander/commands/recordv3.py]: I've added support for legacy records.

File: keepercommander/commands/record_common.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+"""Code/Data shared between recordv3.py and record.py."""
+
+import urllib.parse
+
+import pudb
```
Comment (sk-keeper) [inline keepercommander/commands/record_common.py]: Why this module was introduced into the application?

**Review by sk-keeper — APPROVED** [review]

File: keepercommander/commands/record_common.py (hunk 2 above)
Comment (dstromberg) [reply keepercommander/commands/record_common.py]: As we slacked about, and as the docstring says, this new file has code shared by record.py and recordv3.py.  It has the advantages of avoiding code duplication, avoiding cyclic imports, and being thoroughly statically analyzed.

Comment (dstromberg) [reply keepercommander/commands/record_common.py]: Or do you mean urllib.parse?  Or pudb?  urllib.parse is there to extract data from a URL's query string.  pudb was a mistake; I've removed it.

**Pull request merged**